packages/@aws-cdk/aws-cloudwatch/lib/graph.ts

```diff
@@ -294,6 +295,35 @@ export enum Shading {
   BELOW = 'below'
 }
 
+/**
+ * A set of standard colours that can be used in annotations in a GraphWidget.
+ */
+export enum GraphColor {
```

Comment:
Can we just call this `Color` ?

Also, very interested where you got this set. There are some standard sets of graph colors out there, but is it one of those?

Reply:
BTW this can't be an enum, the implicit conversion Enum -> string only works in JS/TS, not in any of the jsii languages.

Reply:
> There are some standard sets of graph colors out there, but is it one of those?

Yip, from CW dashboards default set on the AWS console. I'm quite happy with their colour palette, so simply stole it.